docs/platforms/apple/common/migration/index.mdx

```diff
@@ -3,7 +3,126 @@ title: Migration Guide
+
+Additionally, the Package.swift sets `swift-tools-version` to 6.0.
+
+### Features Enabled by Default
+
+We now enabled the following features by default:
+
+- **Performance V2 is enabled by default**: The app start duration now finishes when the first frame is drawn instead of when the OS posts the `UIWindowDidBecomeVisibleNotification`. We removed the `enablePerformanceV2` option.
+- **Pre-warmed app start tracking**: The SDK now collects pre-warmed app starts by default via `enablePreWarmedAppStartTracing`.
+- <PlatformLink to="/configuration/app-hangs/#app-hangs-v2">App hang tracking V2</PlatformLink>: This is now the default and we removed the option to enable/disable it.
+
+### Disable App Hang Tracking for Extensions
+
+We now automatically disable app hang tracking for: Widgets, Live Activities, Action Extensions, (Siri) Intent Extensions, Share Extensions. These components run in separate processes or sandboxes with different execution characteristics, which can cause false positive app hang reports.
+
+### Structured Logging
+
+We moved structured logging options out of experimental and made them part of the main API. We added log APIs to `Hub` and `Client`, and logs now include a `sentry.replay_id` attribute.
+
+#### HTTP Client Errors
+
+HTTP client errors now mark sessions as errored. This provides better visibility into failed network requests in your release health data.
+
+### Profiling Changes
+
+We removed all deprecated profiling APIs in version 9.0.0. The only supported profiling method is now <PlatformLink to="/profiling/#enable-ui-profiling">UI Profiling</PlatformLink> (also known as continuous V2 profiling), which was introduced in version 8.49.0.
+
+**Removed deprecated profiling options from `SentryOptions`:**
+- `profilesSampleRate` - used for transaction-based profiling (deprecated)
+- `profilesSampler` - used for transaction-based profiling (deprecated)
+- `enableAppLaunchProfiling` - used for launch profiling (deprecated)
+
+**Migration to UI Profiling:**
+
+If you were using any of the deprecated profiling options, you need to migrate to UI Profiling. Configure profiling by assigning a closure to `SentryOptions.configureProfiling`:
+
+```swift
+import Sentry
+
+SentrySDK.start { options in
+    options.dsn = "___PUBLIC_DSN___"
+    options.configureProfiling = {
+        $0.sessionSampleRate = 1
+        $0.lifecycle = .manual // or .trace for trace-based lifecycle
+    }
+}
+```
+
+For more information, see the <PlatformLink to="/profiling/#enable-ui-profiling">UI Profiling documentation</PlatformLink>.
+
+### Breaking Changes
+
+#### Option Changes
+
+- `enableFileManagerSwizzling` is now a top-level non-experimental option and remains disabled by default.
+- `enableDataSwizzling` is now a top-level option and is enabled by default.
```

Comment:
They're still shown as experimental.enableFileManagerSwizzling and experimental.enableDataSwizzling in code examples.

File affected:
- docs/platforms/apple/common/tracing/instrumentation/automatic-instrumentation.mdx (lines 270, 280, 347-348, 358-359)